test/es6/HTMLComments.js

```diff
@@ -0,0 +1,59 @@
+﻿//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+/* NOTE: This file needs to be treated as binary. It contains mixed line endings, including non-standard
+ *       line endings. Most text editors will not handle the file correctly. If you need to edit this
+ *       file, make sure you do a binary compare to ensure the non-standard line endings have not been lost.
+ *
+ *       'LS' refers to Unicode Character 'LINE SEPARATOR' (U+2028)
+ *       'PS' refers to Unicode Character 'PARAGRAPH SEPARATOR' (U+2029)
+ */
+
+/*
+ * Line terminator sequences - standard (11.3 LineTerminator)
+ */
+ 
+// CRLF
+WScript.Echo("Code before CRLF--> is reachable");
+--> WScript.Echo("Code after CRLF--> is unreachable");
+
+// CR
+WScript.Echo("Code before CR--> is reachable");
--> WScript.Echo("Code after CR--> is unreachable");
+
+// LF
+WScript.Echo("Code before LF--> is reachable");
+--> WScript.Echo("Code after LF--> is unreachable");
+
+// LS u+2028
+WScript.Echo("Code before LS--> is reachable"); --> WScript.Echo("Code after LS--> is unreachable");
+
+// PS u+2029
+WScript.Echo("Code before PS--> is reachable"); --> WScript.Echo("Code after PS--> is unreachable");

```

Comment:
You already gave the unicode sequences in the comment above, so I think this is redundant.